apps/web/src/app/unsubscribe/page.tsx

```
@@ -37,8 +37,8 @@ async function UnsubscribePage({
       <div className=" fixed bottom-10  p-4">
         <p>
           Powered by{" "}
-          <a href="https://unsend.dev" className="font-bold" target="_blank">
-            Unsend
+          <a href="https://usesend.com" className="font-bold" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)